packages/runtime/jest.config.ts

```diff
@@ -1,6 +1,9 @@
 import type { Config } from 'jest'
 
 const config: Config = {
+  moduleNameMapper: {
+    'react-dom/server': '<rootDir>/node_modules/react-dom/server.browser.js',
```

Comment:
Point to the "server.browser" `react-dom/server` implementation for tests to get access to [`renderToReadableStream`](https://react.dev/reference/react-dom/server/renderToReadableStream), instead of much more complex [`renderToPipeableStream`](https://react.dev/reference/react-dom/server/renderToPipeableStream).